.github/workflows/pg-clients.yml

```diff
@@ -42,6 +42,8 @@ jobs:
       github-event-name: ${{ github.event_name }}
 
   build-build-tools-image:
+    permissions:
+      packages: write
```

Comment:
PR descriptions mentions:
```
Grant required packages: read permissions to the workflow
```

and here we grant `write` permissions, what is correct?